README.md

```diff
@@ -207,7 +207,7 @@ Run Rocket.Chat on your easy to use personal device.
 
 # About Rocket.Chat
 
-Rocket.Chat is a Web Chat Server, developed in JavaScript, using the [Meteor](https://www.meteor.com/install) full stack framework.
+Rocket.Chat is a Web Chat Server, developed in JavaScript, using the [Meteor](https://www.meteor.com/developers/install) full stack framework.
```

Comment:
- I find adding /developers/ in link or not directs to same site. 
- I think there is no need of addition.